ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ip-icmp.cfg

```diff
@@ -0,0 +1,9 @@
+!RANCID-CONTENT-TYPE: arista
+!
+ip icmp rate-limit-unreachable 0
+!
+no ip icmp redirect
+!
+ip icmp source-interface Loopback0 vrf TEST
+!
+end
```

Comment:
I could not see `ip-icmp` host being added to the molecule scenario, so was this output generated by molecule or manually? Please add `ip-icmp` to molecule inventory.